control-plane/agents/src/bin/ha/cluster/volume.rs

```diff
@@ -49,10 +49,19 @@ impl VolumeMover {
         mut req: Vec<SwitchOverRequest>,
     ) -> Result<(), anyhow::Error> {
         req.sort_by_key(|r| r.timestamp());
-
+        let mut delayed_enqueue = vec![];
```

Comment:
this seems quite convoluted, why not simply sort? Just impl `Ord` trait for a `SwitchOverRequest` and then simply req.sort() ?

Reply:
Incorporated